### PR TITLE
feat(experimental): add `tui-avatar-labeled`

### PR DESCRIPTION
**Pull request by splincode**
## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring
- [ ] Build or CI related changes
- [ ] Tests related changes
- [ ] Documentation content changes

## What is the new behaviour?

image

### CODE REVIEW

**Review by waterplea — COMMENTED** [review]
1. Let's use `Labeled` with single `l`
2. Let's use component `tui-avatar-labeled` instead of a directive
3. Let's use `gap` instead of `margin-bottom`
4. There shouldn't be a `size` input if all it affects is the width, the width should just rely on the size of the avatar inside of it
5. I wonder if there's a way we can figure out automatically if the text going to take 1 or 2 lines?

File: projects/experimental/components/avatar-labelled/avatar-labeled.styles.less [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    line-height: 0.895rem;
+    font-size: 0.75rem;
+    padding: 0 0.5rem;
+    gap: 0.375rem;
+
+    [tuiFade] {
+        width: calc(100% + 1rem);
+    }
+
+    & *:nth-child(n + 3) {
```
[the remark this answers is not in the document]
Comment (splincode) [reply projects/experimental/components/avatar-labelled/avatar-labeled.styles.less]: fixed

File: projects/experimental/components/avatar-labelled/avatar-labeled.component.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import {ChangeDetectionStrategy, Component, ViewEncapsulation} from '@angular/core';
+
+@Component({
+    selector: 'tui-avatar-labeled',
+    template: '<ng-content></ng-content>',
+    styleUrls: ['./avatar-labeled.styles.less'],
+    encapsulation: ViewEncapsulation.None,
+    changeDetection: ChangeDetectionStrategy.OnPush,
+    host: {
```
[the remark this answers is not in the document]
Comment (splincode) [reply projects/experimental/components/avatar-labelled/avatar-labeled.component.ts:9]: fixed

File: projects/experimental/components/avatar-labelled/avatar-labeled.component.ts [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+        class: 'tui-avatar-labelled',
```
Comment (waterplea) [inline projects/experimental/components/avatar-labelled/avatar-labeled.component.ts:10]: Not needed either

File: projects/experimental/components/avatar-labelled/avatar-labeled.styles.less [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    width: 3.5rem;
+    box-sizing: content-box;
+    flex-direction: column;
+    text-align: center;
+    align-items: center;
+    line-height: 0.895rem;
+    font-size: 0.75rem;
+    padding: 0 0.5rem;
+
+    tui-avatar:first-child {
```
Comment (waterplea) [inline projects/experimental/components/avatar-labelled/avatar-labeled.styles.less:12]: Please remove the `:first-child`, it might not always be the case.

**Review by waterplea — APPROVED** [review]

File: projects/experimental/components/avatar-labelled/avatar-labeled.component.ts (hunk 3 above)
Comment (splincode) [reply projects/experimental/components/avatar-labelled/avatar-labeled.component.ts]: fixed

File: projects/experimental/components/avatar-labelled/avatar-labeled.styles.less (hunk 4 above)
Comment (splincode) [reply projects/experimental/components/avatar-labelled/avatar-labeled.styles.less]: fixed

**Pull request merged**